### PR TITLE
 Added profile to User and added doGet() method to ProfileServlet

### PR DESCRIPTION
**Pull request by huihanlhh**
Working on doPost right now. The code might not be working since I have no means of testing it.

### CODE REVIEW

File: src/main/webapp/WEB-INF/view/profile.jsp [hunk 1]
```diff
@@ -9,74 +9,62 @@
...
+          String profile = UserStore.getInstance().getUser(((User)request.getAttribute("user")).getName()).getProfile();
+        %>
+        <%= profile %>
+      </p>
+
+      <form action="<%= request.getRequestURI() %>" method="POST">
           <div class="form-group">
-            <label class="form-control-label">Title:</label>
-          <input type="text" name="conversationTitle">
+            <h3>Edit your About Me (only you can see this)</h3>
```
Comment (amuhammad135) [inline src/main/webapp/WEB-INF/view/profile.jsp]: Does this edit the profile attribute of user?

Comment (huihanlhh) [reply src/main/webapp/WEB-INF/view/profile.jsp]: It does in my new version. The things up here are pretty old. I will submit again when I'm done with CSS

Comment (amuhammad135) [reply src/main/webapp/WEB-INF/view/profile.jsp]: ok

Comment (amuhammad135) [reply src/main/webapp/WEB-INF/view/profile.jsp]: everything else looks good to me

**Comment by huihanlhh** [comment]
Still have one jsp display issue but I think we should let it go to review

**Pull request merged**